code/datums/components/overlay_lighting.dm

```diff
@@ -250,8 +250,8 @@
 		if(directional)
 			RegisterSignal(new_holder, COMSIG_ATOM_DIR_CHANGE, .proc/on_holder_dir_change)
 	if(overlay_lighting_flags & LIGHTING_ON)
-		make_luminosity_update()
```

Comment:
Please explain the change.

Reply:
moved it below add_dynamic_lumi so it applies the lighting updates after the lights are added, rather than before